eo-maven-plugin/src/main/java/org/eolang/maven/ParseMojo.java

```diff
@@ -190,18 +179,16 @@ private void parse(final ForeignTojo tojo) throws IOException {
             for (final XML error : errors) {
                 Logger.error(
                     this,
-                    "Failed to parse '%s:%s': %s (just logging, because of failOnError=false)",
+                    "Failed to parse '%s:%s': %s",
                     source, error.xpath("@line").get(0), error.xpath("text()").get(0)
                 );
             }
-            if (this.failOnError) {
-                throw new IllegalArgumentException(
-                    String.format(
-                        "Failed to parse %s (%d parsing errors)",
-                        source, errors.size()
-                    )
-                );
-            }
+            throw new IllegalArgumentException(
```

Comment:
@c71n93 I think, we should NOT fail here. We introduced the `verify` goal for this, right? We will fail there. If we fail here, the `verify` mojo will never be called. @maxonfjvipon am I right?

Reply:
@yegor256 you're right, I didn't notice that this code is in `ParseMojo`.
@c71n93 I think we need one more test, that parses with errors, but does not fail